src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,96 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75F;
+    private int size;
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+        this.table = new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
+        checkSize();
+        int position = getPosition(key);
+        Node<K, V> currentNode = table[position];
+        Node<K, V> newNode = new Node<>(key, value, null);
 
+        if (table[position] == null) {
+            table[position] = newNode;
+            size++;
+        } else {
+            while (true) {
+                if (Objects.equals(key, currentNode.key)) {
+                    currentNode.value = value;
+                    return;
+                }
+                if (currentNode.next == null) {
+                    currentNode.next = newNode;
+                    size++;
+                    return;
+                }
+                currentNode = currentNode.next;
+            }
+        }
```

Comment:
generally, using while(true) is not a good practice
```suggestion
	        while (currentNode != null) {
	             if (Objects.equals(key, currentNode.key)) {
	                    currentNode.value = value;
	                    return;
	             }
	             currentNode= currentNode.next;
	        }
	        table[position] = new Node<>(key, value, table[position]);
	        size++;
```